lighthouse-core/config/config.js

```diff
@@ -563,8 +565,13 @@ class Config {
    * @return {!Set<string>}
    */
   static getAuditIdsInCategories(categories) {
-    const audits = _flatten(Object.keys(categories).map(id => categories[id].audits));
-    return new Set(audits.map(audit => audit.id));
+    /** @type {Array<string>} */
```

Comment:
I wanted to try to do some type checking in config, did 2 changes and then scared off by the sea of red. save that for later :)

Reply:
when you say @brendankenny 3 times, your type checking will be resolved automatically

Reply:
yeah, it's the worst file :) We might have to do it a few methods at a time, turning on type checking for changes and then turning it back on when committing so the build stays green :)